hs_core/forms.py

```diff
@@ -23,460 +23,112 @@ class Helper(object):
+                        HTML('{% csrf_token %} '
+                             '<input name="resource-mode" type="hidden" value="edit"/>'
+                             '<div class="modal-header">'
+                             '<button type="button" class="close" '
+                             'data-dismiss="modal" aria-hidden="true">&times;'
+                             '</button>'),
+                        HTML('<h4 class="modal-title" id="myModalLabel"> Add Element </h4>'),
+                        HTML('</div>'
+                             '<div class="modal-body">'
+                             '{% csrf_token %}'
+                             '<div class="form-group">'),
+                        HTML('{% load crispy_forms_tags %} {% crispy add_creator_modal_form %} '),
+                        HTML('</div>'
+                             '</div>'
+                             '<div class="modal-footer">'
+                             '<button type="button" class="btn btn-default" '
+                             'data-dismiss="modal">Close</button>'
+                             '<button type="submit" class="btn btn-primary">'
+                             'Save changes</button>'
+                             '</div>'
+                             '</form>'
+                             '</div>'
+                             '</div>'
+                             '</div>')
                         )
 
-        layout[0] = HTML('<div class="modal fade" id="add-%s-dialog" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">'
-                                    '<div class="modal-dialog">'
-                                        '<div class="modal-content">' % element_name.lower())
-        layout[1] = HTML('<form action="{{ %s.action }}" method="POST" enctype="multipart/form-data"> ' % modal_form_context_name)
-        layout[3] = HTML('<h4 class="modal-title" id="myModalLabel"> {title} </h4>'.format(title=modal_title),)
-        html_str = '{% load crispy_forms_tags %} {% crispy' + ' add_{element}_modal_form'.format(element= element_name.lower()) + ' %}'
+        layout[0] = HTML('<div class="modal fade" id="add-%s-dialog" tabindex="-1" role="dialog" '
+                         'aria-labelledby="myModalLabel" aria-hidden="true">'
+                         '<div class="modal-dialog">'
+                         '<div class="modal-content">' % element_name.lower())
+        layout[1] = HTML('<form action="{{ %s.action }}" method="POST" '
+                         'enctype="multipart/form-data"> ' % modal_form_context_name)
+        layout[3] = HTML('<h4 class="modal-title" id="myModalLabel"> {title} '
+                         '</h4>'.format(title=modal_title),)
+        html_str = '{% load crispy_forms_tags %} {% crispy' + ' add_{element}_modal_form'.format(
+            element=element_name.lower()) + ' %}'
         layout[5] = HTML(html_str)
```

Comment:
@pkdash Is it possible to leverage layout[0] to layout[5] in layout definition above?